src/client/src/OpenFin/Snapshots/Snapshots.tsx

```diff
@@ -1,3 +1,4 @@
+import OpenFin from "@openfin/core"
```

Comment:
funny we did not (seem to) need this before ...
correct tho, we shouldn't be relying on floating "globals" (I know it's "only" typing)
while you are at it, can you just check `fin.d.ts` in client and workspace - I would call that a _tactical_ setting, and would prefer this kind of import-driven approach